swagger.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
make sure swagger detects the root URL too
so that pipelines might be tested easily